pages/qr/[username].js

```diff
@@ -0,0 +1,104 @@
+import Head from 'next/head';
```

Comment:
would it be better to move the `pages/[username].js` file and this one to a single directory `pages/users/`?

Reply:
yes, I like this idea, I will push the changes

Reply:
fixed